cmd/test.go

```diff
@@ -384,17 +391,21 @@ func (t *Test) GetCmd() *cobra.Command {
 			}
 
 			path += "/keploy"
+			t.logger.Info("", zap.Any("keploy test and mock path", path))
 
-			testReportPath := path + "/testReports"
+			testReportPath := ""
 
-			testReportPath, err = pkg.GetNextTestReportDir(testReportPath, models.TestRunTemplateName)
-			if err != nil {
-				t.logger.Error("failed to get the next test report directory", zap.Error(err))
-				return err
+			if generateTestReport {
+				testReportPath = path + "/testReports"
+	
+				testReportPath, err = pkg.GetNextTestReportDir(testReportPath, models.TestRunTemplateName)
+					t.logger.Info("", zap.Any("keploy testReport path", testReportPath))
+					if err != nil {
+						t.logger.Error("failed to get the next test report directory", zap.Error(err))
+						return err
+					}
 			}
-
-			t.logger.Info("", zap.Any("keploy test and mock path", path), zap.Any("keploy testReport path", testReportPath))
```

Comment:
Why have these essential logs been removed?

Reply:
Similar reason divided it into two different logs